### DISCUSSION TITLE
new problem url types

### DISCUSSION THREAD

**Discussion by bpglobalnetworks** [AzuraCast/AzuraCast#5568] [Q&A]
the old urls work fine, but the new ones don't work at all, only one of four could work, why is this problem? I am working on the stable version

This message appears when I access the new links : The file you requested could not be found

**Comment by paulcjkp** [comment]
Can we get a little more information?  What are the old URLs vs new URLs?  Which version of the stable release are you on?  And if you can paste the relevant log files here as well, that would be more helpful.
